.github/workflows/run-tests.yml

```diff
@@ -487,12 +487,24 @@ jobs:
           annotate_only: true
           skip_annotations: true
 
+      - name: Get job ID from workflow run
+        id: get_job_id
+        run: |
+          set -e
+          job_id=$(curl -L \
+            -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{secrets.GITHUB_TOKEN}}" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            "https://api.github.com/repos/${GITHUB_REPOSITORY}/actions/runs/${GITHUB_RUN_ID}/jobs" \
+            | jq "first(.jobs[] | select(.name | endswith(\"$job_name\")) | .id)")
```

Comment:
The `job_name` variable is unset. I believe that's why we're getting the same job ID `34400904242` for every job:

```
Artifact junit-xml--12324062875--34400904242--1--ubuntu-20.04--postgres12--1--functional-test has been successfully uploaded!
```

(Could do `set -u` to catch unset bash variables)

Reply:
yeah, going to try few more approaches.